RecoTracker/LSTCore/interface/SegmentsSoA.h

```diff
@@ -15,6 +15,15 @@ namespace lst {
                       SOA_COLUMN(FPX, dPhiChanges),
                       SOA_COLUMN(FPX, dPhiChangeMins),
                       SOA_COLUMN(FPX, dPhiChangeMaxs),
+#ifdef CUT_VALUE_DEBUG
```

Comment:
I forget how these flags are coded, but is there any way to make these separate compile-time flags, like SG_CUT_DEBUG, MD_CUT_DEBUG, etc. that are all triggered by the same -d option? This way people could disable flags for different objects if they aren't needed to help mitigate timing increases.